patches/gnustep-gui-private.patch

```diff
@@ -831,19 +831,10 @@ index d3bf78e6b..052ed12c8 100644
  + (NSGraphicsContext *) graphicsContextWithAttributes: (NSDictionary *)attributes;
```

Comment:
Wouldn’t it be better to set up an windows-msvc branch for libs-gui with all your changes? That way they could already be run through the CI as well to make sure nothing else breaks, and the build script here could just point to that branch.

Reply:
I would rather have patches against master instead of some arbitrary branch

Reply:
Also, some of these changes should make it into master.

Reply:
Hopefully all of them! (Conditionally if needed.)

Reply:
Is there a define that is set by msvc that we can use in some cases to conditionally compile things?   If there isn't one we should define one.

Reply:
Yeah, the Microsoft compiler defines `_MSC_VER`. See e.g. here:
https://github.com/gnustep/libs-base/blob/6657796e1599d57410a6e837005216232e9f440a/Source/NSFileManager.m#L78